src/test/migration.ts

```diff
@@ -2,8 +2,8 @@ import type { Knex } from "knex";
 
 import type { DatabaseDialect } from "../dialects/index.js";
 
-const isMySqlDialect = (dialect: DatabaseDialect) =>
-  dialect === "mysql" || dialect === "mysql2";
+export const isMySqlDialect = (dialect: DatabaseDialect) =>
```

Comment:
You've changed dialect and service here? probably want to make a choice. DB_TYPE is my sentiment.

Reply:
I removed `msyql2` dialect since it's really a client library rather than a database dialect. The only options now are `postgres | mysql | mariadb`. I'd rather not stray too far from the original `adminjs-sql` library if possible.